core/cmd/client.go

```diff
@@ -695,9 +695,9 @@ func (t *promptingAPIInitializer) Initialize(orm sessions.ORM) (sessions.User, e
 			email := t.prompter.Prompt("Enter API Email: ")
 			pwd := t.prompter.PasswordPrompt("Enter API Password: ")
 			// On a fresh DB, create an admin user
-			user, err := sessions.NewUser(email, pwd, sessions.UserRoleAdmin)
-			if err != nil {
-				t.lggr.Errorf("Error creating API user: ", err, "err")
+			user, err2 := sessions.NewUser(email, pwd, sessions.UserRoleAdmin)
```

Comment:
Maybe a good time to ask why we have this shadowing lint rule enabled. In my experience, it's never been an issue, and wherever we've had to satisfy it, it hasn't really added any value to the code, at least not that I'm seeing.

Reply:
IIRC it is enabled by default.
I don't have a strong opinion, but I think the rationale is that shadowing can be error prone by leading to unintended consequences in future edits. I suppose it is more readable this way since when you see a new name you know it must be a new variable, but when the name is reused you could mistakenly assume it's the same var.

Reply:
And this case is a poor example, since the assignment is right there on the previous line - but it could be arbitrarily far away, and wouldn't need to be far at all to be clipped off in a PR review on github. :shrug:

Reply:
An example of this being an issue is in `chainlink/core/scripts/chaincli/handler/keeper_upkeep_history.go` that I also fixed in this PR. At the end of the `if` and the `else` blocks there was an assignment to `err` that was checked outside off the block, or that was the intention. Actually, the assignment was to an `err` that shadowed the original `err` and the check would always indicate no error, even if there was one.